pkg/drivers/hyperv/hyperv_windows.go

```diff
@@ -373,3 +376,34 @@ func (d *Driver) GetIP() (string, error) {
 
 	return resp[0], nil
 }
+
+func (d *Driver) GetSharedDirs() ([]drivers.SharedDir, error) {
+	dirs := []drivers.SharedDir{}
+	for _, dir := range d.SharedDirs {
+		if !smbShareExists(dir.Tag) {
+			return []drivers.SharedDir{}, drivers.ErrNotSupported
+		}
+
+		dir.Target = convertToUnixPath(dir.Target)
+		dir.Type = "cifs"
+		dir.Tag = smbShareUNCPathFromTag(dir.Tag)
+
+		dirs = append(dirs, dir)
+	}
+	return dirs, nil
+}
+
+func convertToUnixPath(path string) string {
+	// converts a path like c:\users\crc to /mnt/c/users/crc
```

Comment:
A comment explaining why we do this would be very useful, as it's non obvious why we don't convert this to /users/crc as for macOS. This comment could link to https://github.com/containers/podman/blob/67c4068bb35fd1aad95b1701c94ed11183d0fd66/pkg/specgen/winpath.go#L23-L41

Reply:
Thanks, i did not look at this code, only noticed the behaviour that it appends the `/mnt`